src/unix/linux_like/linux/mod.rs

```diff
@@ -1133,7 +824,8 @@ s! {
             target_arch = "aarch64",
             target_arch = "riscv64",
             target_arch = "riscv32",
-            target_arch = "loongarch64"
+            target_arch = "loongarch64",
+            target_os = "emscripten"
         ),
         repr(align(4))
```

Comment:
Why emscripten config being adjusted here? Happens a few other places in this file

Reply:
ah, that was a leftover of the commonization that I reversed. fixed